tests/lib/collectors/grr_hunt.py

```diff
@@ -335,5 +333,90 @@ def testGetAndWriteWrongResults(self, mock_list_results):
+        reason='random reason',
+        hunt_description='random description',
+        grr_server_url='http://fake/endpoint',
+        grr_username='admin',
+        grr_password='admin',
+        approvers='approver1,approver2',
+        verify=False,
+        match_mode='all',
+        client_operating_systems='win,linux',
+        client_labels='label1',
+        client_limit=999,
+        process_ignorelist=['\\.exe', 'ignore1'],
+    )
+
+  def testInitialization(self):
+    """Tests that the collector is correctly initialized."""
+    runner_args = self.grr_hunt_yara.hunt_runner_args
+    self.assertEqual(runner_args.client_rule_set.match_mode, runner_args.client_rule_set.MATCH_ALL)
+    self.assertTrue(runner_args.client_rule_set.rules[0].os.os_windows)
+    self.assertEqual(runner_args.client_rule_set.rules[1].label.label_names, ['label1'])
+    self.assertEqual(runner_args.client_limit, 999)
+    self.assertEqual(runner_args.avg_cpu_seconds_per_client_limit, 2000)
+
+  @mock.patch('grr_api_client.api.GrrApi.CreateHunt')
+  def testProcess(self, mock_CreateHunt):
+    self.test_state.StoreContainer(
+      containers.YaraRule(
+        name='test',
+        rule_text=('rule test { strings: $a = "abcdefg" condition: '
+                   '$a and pe.DLL }')))
+
+    self.test_state.StoreContainer(
+      containers.YaraRule(
+        name='test2',
+        rule_text=('rule test { strings: $a = "0123456" condition: '
+                   '$a and math.entropy($a) }')))
+
+    expected_runner_args = flows_pb2.HuntRunnerArgs(
+      description='random description'
+    )
+    expected_runner_args.client_rule_set.match_mode = expected_runner_args.client_rule_set.MATCH_ALL
+    rule = expected_runner_args.client_rule_set.rules.add()
+    rule.rule_type = rule.OS
+    rule.os.os_windows = True
+    rule.os.os_linux = True
+    rule.os.os_darwin = False
+
+    rule = expected_runner_args.client_rule_set.rules.add()
+    rule.rule_type = rule.LABEL
+    rule.label.label_names.append('label1')
+
+    expected_runner_args.client_limit = 999
+    expected_runner_args.client_rate= 1000
+    expected_runner_args.crash_limit = 1000
+    expected_runner_args.per_client_cpu_limit = 2000
+    expected_runner_args.avg_cpu_seconds_per_client_limit = 2000
+    expected_runner_args.network_bytes_limit = 10_737_418_240
+
+    self.grr_hunt_yara.Process()
+    mock_CreateHunt.assert_called_with(
+      flow_name='YaraProcessScan',
+      flow_args=flows_pb2.YaraProcessScanRequest(
+          yara_signature=('import "pe"\nimport "math"\n\nrule test { '
+                          'strings: $a = "abcdefg" condition: $a and pe.DLL }'
+                          '\n\nrule test { strings: $a = "0123456" condition:'
+                          ' $a and math.entropy($a) }'),
+          ignore_grr_process=True,
+          process_regex=r"(?i)^(?!\.exe|ignore1).*",
+          dump_process_on_match=True,
+          process_dump_size_limit= 268_435_456,
```

Comment:
what is this 268_435_456? is it mean tot be an int?

Reply:
`_` are a way to separate thousands in integers. so it's equivalent to `268435456` but just easier to read (supposedly)